Document that Ruby profiler depends on pkg-config (#16632)
DataDog/documentation: content/en/profiler/enabling/ruby.md

- Linux (GNU libc) x86-64, aarch64
- Alpine Linux (musl libc) x86-64, aarch64

> You will also need either the [`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) or the [`pkgconf`](https://github.com/pkgconf/pkgconf) Linux system utility installed.

Reviewer: ```suggestion
Yo also need either the [`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) or the [`pkgconf`](https://github.com/pkgconf/pkgconf) Linux system utility installed.
```

> This utility is available on the software repositories of most Linux distributions. You will find it on:

Reviewer: ```suggestion
This utility is available on the software repositories of most Linux distributions. For example:
```

> - The `pkg-config` package on [Homebrew](https://formulae.brew.sh/formula/pkg-config) and [Debian](https://packages.debian.org/search?keywords=pkg-config)/[Ubuntu](https://packages.ubuntu.com/search?keywords=pkg-config)-based Linux

Reviewer: ```suggestion
- The `pkg-config` package is available for [Homebrew](https://formulae.brew.sh/formula/pkg-config) and [Debian](https://packages.debian.org/search?keywords=pkg-config) and [Ubuntu](https://packages.ubuntu.com/search?keywords=pkg-config)-based Linux
```

> - The `pkgconf` package on [Arch](https://archlinux.org/packages/?q=pkgconf) and [Alpine](https://pkgs.alpinelinux.org/packages?name=pkgconf)-based Linux

Reviewer: ```suggestion
- The `pkgconf` package is available for [Arch](https://archlinux.org/packages/?q=pkgconf) and [Alpine](https://pkgs.alpinelinux.org/packages?name=pkgconf)-based Linux
```

> - The `pkgconf-pkg-config` package on [Fedora](https://packages.fedoraproject.org/pkgs/pkgconf/pkgconf-pkg-config/)/[Red Hat](https://rpmfind.net/linux/rpm2html/search.php?query=pkgconf-pkg-config)-based Linux

Reviewer: ```suggestion
- The `pkgconf-pkg-config` package is available for [Fedora](https://packages.fedoraproject.org/pkgs/pkgconf/pkgconf-pkg-config/) and [Red Hat](https://rpmfind.net/linux/rpm2html/search.php?query=pkgconf-pkg-config)-based Linux
```
